db/database.go

```diff
@@ -1749,11 +1757,12 @@ func (context *DatabaseContext) UseViews() bool {
 	return context.Options.UseViews
 }
 
+// TOR: This function is duplicated on DatabaseCollection
```

Comment:
It sounds like we want a consistent pattern for how to access DatabaseContext configuration from a DatabaseCollection.  Need to decide whether it's going to be a duplicated helper function on DatabaseCollection, or whether callers will need to do databaseCollection.dbCtx.DeltaSyncEnabled().  The former makes the code less wordy and means less refactoring, but the latter makes the code a self-documenting that these properties aren't collection-specific.  I'm not convinced one way or the other, but we should pick an approach and apply it consistently.